bin/validate-templates.sh

```diff
@@ -10,7 +10,6 @@ readonly output_path="/tmp/otomi/generated-crd-schemas"
 readonly schemas_bundle_file="$output_path/all.json"
 readonly k8s_resources_path="/tmp/otomi/generated-manifests"
 readonly jq_file=$(mktemp -u)
-readonly cluster_env=$(cluster_env)
```

Comment:
why is this not correct? does this script not get invoked via `otomi validate-templates`? Then stay DRY and reuse

Reply:
did you check that is is working as intended @Morriz ?